### DISCUSSION TITLE
How are we going to create the multiple branches?

### DISCUSSION THREAD

**Discussion by TafeidMatt** [Kade03/Tafe22S203#3] [General]
![image](https://user-images.githubusercontent.com/112943875/189531175-30cf47f4-12e0-45a1-8e5e-888fde79f1a8.png)

We need a release, dev branch; team leader may need to make this.
We will need feature 1.1, 1.2 and 1.3 branches, we will probably make each ourselves.

All of our features branches need to come from the dev branch, 
then when complete merge back to the dev branch>release>master v1.0.0.

**Comment by Kade03** [comment]
so ive made the development branch for the github repos, im pretty sure the assignment sheet says that each team member needs to create there own feature branch.

**Reply by TafeidMatt** [reply]
Lets do that.

**Reply by WillLeslie03** [reply]
which branch do you want me to crete?

**Reply by Kade03** [reply]
im doing mortgage calc feature i think you said you were wanting to do the currency calc yeah?
